tests/cdc_test.cc

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2019 ScyllaDB
+ */
+
+/*
+ * This file is part of Scylla.
+ *
+ * Scylla is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Scylla is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <seastar/testing/thread_test_case.hh>
+
+#include "tests/cql_test_env.hh"
+
+SEASTAR_THREAD_TEST_CASE(test_with_cdc_parameter) {
```

Comment:
Oh, I see that later commits add more checks to this one test... So at the end it tests more than it tests here... Maybe you can say that in the commit message.

Reply:
I added commit message. Regarding testing persistence of the parameter - I haven't found a way to do it with cql_test_env.

Reply:
We could add to cql_test_env a feature where we can shut down the test environment without removing the data files, and then bring up the test environment again - with the same data files. But this is beyond the scope of this patch. And in any case I think what we really need isn't a more advanced cql_test_env but rather dropping it completely, and replacing it by functional tests in Python, similar to what we did in Alternator (and similar in some respects to what @kostja has also been suggesting). But since it's beyond the scope of this patch set, I'll mark the discussion resolved.